community/tcl/build

```diff
@@ -1,14 +1,21 @@
 #!/bin/sh -e
 
-IFS=. read -r ver1 ver2 _ < "${0%/*}/version"
+IFS=. read -r maj min _ < "${0%/*}/version"
 
-unix/configure \
+rm -rf pkgs/sqlite3*
```

Comment:
This can be removed if `--system-sqlite` is used no?

Reply:
based on other distros' build files and the source directory itself, it's a bit hard to tell if `--with-system-sqlite` is still an option in 8.6.10, so this is just forcing it.